test/test.js

```diff
@@ -21289,9 +21357,13 @@
       }, 192);
 
       setTimeout(function() {
+        assert.strictEqual(callCount, 1);
```

Comment:
Once I fixed the other bug, this test started failing.  After reviewing the test, it seems like it had some bad assertions.  Or maybe I am misunderstanding the way throttle works.

What do you think?